pkg/api/folder.go

```diff
@@ -695,12 +700,13 @@ func (fk8s *folderK8sHandler) createFolder(c *contextmodel.ReqContext) {
 		return
 	}
 
-	f, err := internalfolders.UnstructuredToLegacyFolderDTO(*out)
+	folderDTO, err := fk8s.newToFolderDto(c, *out)
```

Comment:
Don't we still need the functionality from `UnstructuredToLegacyFolderDTO`? 

One options would be to combine `newToFolderDto` and `UnstructuredToLegacyFolderDTO`. It could be called `UnstructuredToLegacyFolderDTO` and we could keep it in conversions. WDYT?

Reply:
Never mind, I just saw that newToFolderDto calls UnstructuredToLegacyFolderDTO 👍